src/models/timeaxisheaderview.h

```diff
@@ -0,0 +1,30 @@
+#ifndef TIMEAXISHEADERVIEW_H
+#define TIMEAXISHEADERVIEW_H
+
+#include <QHeaderView>
+
+#include "data.h"
+
+class FilterAndZoomStack;
+
+class TimeAxisHeaderView : public QHeaderView
+{
+    Q_OBJECT
+public:
+    explicit TimeAxisHeaderView(FilterAndZoomStack* filterAndZoomStack, Qt::Orientation orientation,
```

Comment:
the orientation should always be horizontal, never vertical - so no need to pass it along

Reply:
Header texts and sort indicators are restored. The layout of the ticks and the texts is improved.